spec/rails_helper.rb

```diff
@@ -11,7 +11,6 @@
 
 # auto-require all ruby files in the support directory
 Dir[Rails.root.join('spec/support/*.rb')].each { |f| require f }
-require 'cocina/rspec'
```

Comment:
Can you say more about shifting this line? Not a blocker, just curious.

Reply:
It has to come after `config.include FactoryBot::Syntax::Methods` to use cocina factories.

Reply:
Makes sense. Worth a comment?